### DISCUSSION TITLE
Astrotomic/laravel-translatable

### DISCUSSION THREAD

**Discussion by anatolyeltsov** [filamentphp/filament#3103] [Help]
I've created a model with translatable name and a simple Filament resource but when I click on edit or delete buttons on the list page I get an error:

`App\Models\Test::getAttributeAndLocale(): Argument #1 ($key) must be of type string, null given`

Am I missing something that I need to add in actions?

Also what's the best way to get translations on non-simple resource edit page?
Do I need to manually modify data before passing it to the form (let's say I'd like to have name:en, name:de, etc form fields)?

Thanks!

**Comment by Oleksandr-Moik** [comment]
Hi! This error happens when specified the resource [title attribute](https://filamentphp.com/docs/3.x/panels/resources/getting-started#record-titles) is translatable. 

To fix this error, specify the locale in the title, for [example](https://github.com/CactusGalaxy/GalaxyStoreExample/blob/main/app/Filament/Resources/CategoryResource.php#L43):

```php
protected static ?string $recordTitleAttribute = 'name:en';
```

-----

Also for pages with a translatable model you can use package - https://github.com/CactusGalaxy/FilamentAstrotomic
